WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -49,13 +54,6 @@ extension NewOrderHostingController {
     override func shouldPopOnSwipeBack() -> Bool {
         return shouldPopOnBackButton()
     }
-
-    private func presentDiscardChangesActionSheet() {
```

Comment:
After discarding the changes we can either dismiss or pop the view, so to avoid duplicating the code [I've created a method](https://github.com/woocommerce/woocommerce-ios/blob/b8486b449baf1384ebcf3d059785236ad12e18a2/WooCommerce/Classes/ViewRelated/Orders/Order%20Creation/NewOrder.swift#L83) that presents the action sheet and received the `onDiscard` closure as a param.

Reply:
Good question! TBH although we only have two ways of closing this flow, we have three "triggers"; the back button (pop), the cancel button (dismiss) and the swipe down (dismiss), so we could leave them to not to do the same two times (cancel and swipe). But it's true that we could have only one method for both triggers, something like `presentDiscardChangesActionSheetAndDismissOnDiscard` 🤔 You know what I mean?

That said, I don't have a strong preference on this, so I might as well revert the changes 🙂

Reply:
> so we could leave them to not to do the same two times (cancel and swipe)

This makes sense 👍 Let's keep the changes then!